webrtc.html

```diff
@@ -1816,9 +1795,20 @@
                         </ol>
                       </li>
                       <li>
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        equal to any ufrag present in the corresponding
+                        <a>media description</a> of an applied remote
+                        description, reject <var>p</var> with an
```

Comment:
Have to specify whether this is current description, pending description, or both. This matters in case of rollback (the issue Peter raised).